pages/index.tsx

```diff
@@ -81,6 +81,31 @@ export default function Home() {
         return response
     }
 
+    const downloadImage = () => {
```

Comment:
Please ensure that the downloaded image is in jpg or png type so that majority of users can easily view it.

Reply:
I have made the changes now user can download the images in jpg or png format

<img width="255" alt="Screenshot 2023-10-01 160349" src="https://github.com/SDC-USAR/ArtiGenius-AI/assets/108524706/144cb0ce-4ce5-40e3-8724-56e79eacbdd6">